usr/echod/echod.c

```diff
@@ -15,6 +11,9 @@
 
 #define DEFAULT_PORT 7777
 
+/// \file usr:echod:echod.c
+/// \brief The file implements UDP echo server
+
```

Comment:
This is an example application. I think we should document only kernel APIs, and leave this as-is.